src/utils/authentication/aad.strategy.ts

```diff
@@ -79,7 +79,7 @@ export class AzureADStrategy
     const upstreamAccessToken = await this.getCachedBearerToken();
     const response = await this.getDownstreamAccessToken(upstreamAccessToken);
     const downstreamAccessToken = response['access_token'] as string;
-    console.info(`Downstream access token: ${downstreamAccessToken}`);
```

Comment:
If you don't want to see info messages you can control this in the bootstrap method in main.ts. No need to remove them.